core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -395,6 +415,61 @@ object ConsumerGroupCommand extends Logging {
       result
     }
 
+    def deleteOffsets(): (Option[Throwable], Map[TopicPartition, Throwable]) = {
```

Comment:
We could consider using the `Either` type here.

Reply:
I will check it. I have never used it.

Reply:
I have refactored the code and it is not necessary anymore.